src/modules/week/calendar-week-view-hour-segment.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, Input, TemplateRef } from '@angular/core';
+import { WeekViewHourColumn } from 'calendar-utils';
+
+@Component({
+  selector: 'mwl-calendar-week-view-hour-segment',
+  template: `
+    <ng-template
+      #defaultTemplate
+      let-segment="segment"
+      let-locale="locale"
+      let-segmentHeight="segmentHeight"
+      let-isTimeLabel="isTimeLabel">
+      <div
+        class="cal-hour-segment"
+        [style.height.px]="segmentHeight"
+        [class.cal-hour-start]="segment.isStart"
+        [class.cal-after-hour-start]="!segment.isStart"
+        [ngClass]="segment.cssClass">
+        <div class="cal-time" *ngIf="isTimeLabel">
+          {{ segment.date | calendarDate:'weekViewHour':locale }}
```

Comment:
weekViewHour with locale "de" will make the time segment format like this:
<img width="543" alt="timesegmentformat" src="https://user-images.githubusercontent.com/4215561/43714103-330693ba-997c-11e8-9e49-ffffd180ab89.png">

but providing a customHourSegmentTemplate changes it to this:
 
<img width="935" alt="customhoursegmenttemplate" src="https://user-images.githubusercontent.com/4215561/43714376-12119b5e-997d-11e8-8a80-7f5ef0386077.png">

I don't think this is the wished behavior?